.buildkite/rest-tests.yaml

```diff
@@ -1,36 +1,12 @@
 ---
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 steps:
-  - label: ":elasticsearch: :python: ES Serverless ({{ matrix.python }}/{{ matrix.connection_class }}) Python Test Suite: {{ matrix.suite }}"
-    agents:
-      provider: gcp
-    env:
-      PYTHON_VERSION: "{{ matrix.python }}"
-      PYTHON_CONNECTION_CLASS: "{{ matrix.connection_class }}"
-      # TEMPORARY for 3.11
-      # https://github.com/aio-libs/aiohttp/issues/6600
-      AIOHTTP_NO_EXTENSIONS: 1
-      # https://github.com/aio-libs/frozenlist/issues/285
-      FROZENLIST_NO_EXTENSIONS: 1
-      # https://github.com/aio-libs/yarl/issues/680
-      YARL_NO_EXTENSIONS: 1
-      EC_REGISTER_BACKEND: "appex-qa-team-cluster"
-      EC_ENV: "qa"
-      EC_REGION: "aws-eu-west-1"
-      EC_PROJECT_PREFIX: "esv-client-python-test-{{ matrix.python }}-{{ matrix.connection_class }}"
-    matrix:
-      setup:
-        python:
-          - "3.7"
-          - "3.8"
-          - "3.9"
-          - "3.10"
-          - "3.11"
-        connection_class:
-          - urllib3
-          - requests
-    command: echo "Skipping ./.buildkite/run-tests for now"
-    artifact_paths: "junit/*-junit.xml"
+  - label: ":pipeline: Generate steps"
+    command: |
+      set -eo pipefail
+
+      python3 -m pip install pyyaml
+      python3 .buildkite/generatesteps.py | buildkite-agent pipeline upload
```

Comment:
Since we need two builds per combination (run-tests and teardown), the matrix functionality does not work and we need to generate the steps.